HopsanGenerator/src/generators/HopsanFMIGenerator.cpp

```diff
@@ -401,7 +401,10 @@ bool HopsanFMIGenerator::generateFromFmu1(const QString &rFmuPath, const QString
         {
             addConstants.append("        ");
         }
-        addConstants.append("addConstant(\""+parNames.at(i)+"\", \"\", \"\", 0, "+parVars.at(i)+");\n");
+
+        fmi1_import_variable_t *pVar = fmi1_import_get_variable_by_name(fmu, parNames.at(i).toStdString().c_str());
+        double startValue = fmi1_import_get_real_variable_start(fmi1_import_get_variable_as_real(pVar));
```

Comment:
Import crash here, when import parameter is not of type double

Reply:
I made all parameter types supported for FMI1 import/export, and update the testmodel.